Assets/Python/Plague.py

```diff
@@ -146,72 +141,62 @@ def stopPlague(self, iPlayer):
 
 
 	def processPlague(self, iPlayer):
-
 		pPlayer = gc.getPlayer(iPlayer)
 		#first spread to close locations
 		cityList = [] #see below
 		lInfectedCities = []
 		lNotInfectedCities = []
 		for city in utils.getCityList(iPlayer):
 			cityList.append(city) #see below
-			if (city.hasBuilding(iPlague)):
+			if city.hasBuilding(iPlague):
 				lInfectedCities.append(city)
 				#print ("plague in city", city.getName())
-				#if (pPlayer.isHuman()):
+				#if pPlayer.isHuman():
 				#	CyInterface().addMessage(iPlayer, True, iDuration/2, CyTranslator().getText("TXT_KEY_PLAGUE_PROCESS_CITY", ()) + " " + city.getName(), "AS2D_PLAGUE", 0, "", ColorTypes(iLime), -1, -1, True, True)

```

Comment:
I think you can also outright remove all the commented code. We still have it in git if we ever want to bring something back.